platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/EnhancedKeyStoreLoader.java

```diff
@@ -309,6 +330,29 @@ public EnhancedKeyStoreLoader scan() throws KeyLoadingException, KeyStoreExcepti
         return this;
     }
 
+    @Nullable
+    private BlsPrivateKey resolveTssPrivateKey(@NonNull final NodeId nodeId, @NonNull final String nodeAlias)
+            throws KeyLoadingException {
+        Objects.requireNonNull(nodeId, MSG_NODE_ID_NON_NULL);
+        Objects.requireNonNull(nodeAlias, MSG_NODE_ALIAS_NON_NULL);
+
+        Path keyLocation = keyStoreDirectory.resolve(String.format("t-private-%s.tss", nodeAlias));
```

Comment:
could validate or sanitize `nodeAlias` to ensure it is always safe for filenames before using it to construct the path.

Reply:
Will leave as is like other uses of nodeAlias